addons/web/static/tests/views/list/list_view.test.js

```diff
@@ -5314,6 +5314,12 @@ test(`duplicate all records`, async () => {
     await contains(`.o_cp_action_menus .dropdown-toggle`).click();
     await toggleMenuItem("Duplicate");
 
+    // A confirmation dialog should appear when duplicating multiple records.
+    expect(`.modal`).toHaveCount(1, {
+        message: "Are you sure that you want to duplicate all the selected records?",
+    });
```

Comment:
It looks like you try to assert the content of the dialog, but this is not what this code does.
```suggestion
    expect(`.modal`).toHaveCount(1);, {
    expect(`.modal .modal-body`).toHaveText("Are you sure that you want to duplicate all the selected records?");
```